src/socket.c

```diff
@@ -3300,6 +3300,20 @@ static int handle_socket_input(const char *simbuffer, int simlen, const char *en
     if (xsock->constate <= SS_CONNECTING || xsock->constate >= SS_ZOMBIE)
 	return 0;
 
+   /* We assumed last text was a prompt, but now we have more text, so
```

Comment:
@warmstarter That block was removed but I think that it may be valid..

Reply:
Ok, so was there a reason for adding this back in? You said with the other one that it fixed all of your problems. Other than the fact that it involves the word 'prompt' was there any known reason to add this code back in? I'd have to look to into it more, but I presume there was likely a reason it was initially removed and if you have no actual reason to add it back in, I don't see why you would?

Reply:
I'm still wary of this code, as I can understand that noting there were prompt issues you were looking for prompt related code. However, just randomly adding this back in doesn't seem like it has benefit.  I've been going through the TF codebase for awhile now, and there's plenty of code in it that I've found that is no longer connected to anything. If removing this code didn't introduce problems and adding it didn't fix them, I'm not sure I see the reason.

Reply:
The confusion here is because your fork lacks this patch:
https://github.com/Sketch/tinyfugue/commit/d3cd4089e95458b6459ae453f3c802e6dfe01416

I'd also recommend changing the varlist.h from US-ASCII to UTF-8 as UTF-8 is backwards compatible. I can't think of a situation where you wouldn't want this to be UTF-8 on the client side.